### ISSUE TITLE
DropDown.Button缺少getPopupContainer属性

### ISSUE THREAD

**Issue by blankzust** [ant-design/ant-design#6461]
### What problem does this feature solve?
和DropDown上有getPopupContainer属性的效果一样，有时候会出现滚动后位置失调的问题

### What does the proposed API look like?
<DropDown.Button  getPopupContainer={() =>document.getElementById("content")} overlay={ menu } trigger={['click']} onClick={ mergeFunc }/>

**Comment by benjycui** [comment]
Yep, we need to extract `getPopupContainer ` [here](https://github.com/ant-design/ant-design/blob/81be11534c9204048676cd07a943f8f40a47e114/components/dropdown/dropdown-button.tsx#L32-L35) and pass it to `Dropdown`.

**Comment by benjycui** [comment]
@blankzust could you PR to fix it?

**Comment by blankzust** [comment]
Ok, I will try it.

**Comment by benjycui** [comment]
@blankzust  You can simplify TypeScript definition at the same time https://github.com/ant-design/ant-design/pull/6482

**Comment by blankzust** [comment]
I have added the props.But it didn't work.I want to see where do the getPopupContainer works.Could you please tell me @benjycui 
<pre>
<code>
import React from 'react';
import Button from '../button';
import Icon from '../icon';
import Dropdown from './dropdown';
const ButtonGroup = Button.Group;
import classNames from 'classnames';

export interface DropdownButtonProps {
  prefixCls?: string;
  className?: string;
  type?: 'primary' | 'ghost' | 'dashed';
  onClick?: React.MouseEventHandler<any>;
  trigger?: ('click' | 'hover')[];
  align?: any;
  getPopupContainer?: (triggerNode: Element) => HTMLElement;
  overlay: React.ReactNode;
  visible?: boolean;
  disabled?: boolean;
  onVisibleChange?: (visible: boolean) => void;
  style?: React.CSSProperties;
  children?: any;
  placement?: 'topLeft' | 'topCenter' | 'topRight' | 'bottomLeft' | 'bottomCenter' | 'bottomRight';
}

export default class DropdownButton extends React.Component<DropdownButtonProps, any> {
  static defaultProps = {
    placement: 'bottomRight',
    type: 'default',
    prefixCls: 'ant-dropdown-button',
  };

  render() {
    const {
      type, overlay, trigger, align, children, className, onClick, prefixCls,
      disabled, visible, onVisibleChange, placement, getPopupContainer, ...restProps,
    } = this.props;
    const cls = classNames(prefixCls, className);

    const dropdownProps = {
      align,
      overlay,
      trigger: disabled ? [] : trigger,
      onVisibleChange,
      placement,
    };

    if ('visible' in this.props) {
      (dropdownProps as any).visible = visible;
    }

    return (
      <ButtonGroup {...restProps} className={cls}>
        <Button type={type} onClick={onClick} disabled={disabled}>{children}</Button>
        <Dropdown {...dropdownProps}>
          <Button type={type} disabled={disabled}>
            <Icon type="down" />
          </Button>
        </Dropdown>
      </ButtonGroup>
    );
  }
}
</code>
</pre>

**Comment by blankzust** [comment]
I am very interested in your code.^-^

**Comment by blankzust** [comment]
It works.

**Comment by benjycui** [comment]
> I am very interested in your code.^-^

You can try to propose more PRs, then you will know more about our code while trying to modify it.

See: https://github.com/ant-design/ant-design/issues?q=is%3Aissue+is%3Aopen+label%3A%22PR+welcome%22
